### ISSUE TITLE
Request that rapids don't delay redocking of boat rides

### ISSUE THREAD

**Issue by jamtraks** [OpenRCT2/OpenRCT2#17447]
### Operating System

Windows 11 64-bit

### OpenRCT2 build

v0.4.0, c6302a1 on master

### Describe the issue

I have built a few parks with boat rides, and the rapids in the water where the boats roam, are blocking the boats from redocking. The guests can't get off of the boat rides and are complaining.
I want the rapids to be shown in the water but i also want the boats to be able to re dock.
I think this variable should be completely removed from the game. If not, it should be a feature that can be turned on or off, the fact that the rapids block the boats from redocking.
Over all i think this variable  is absolutely  silly, and in real life this wouldn't have anything to do with boats redocking or not at an amusement park.
Would this ruin any park or would anyone miss the blocking of boat redocking from rapids?
Is it removable from the game?

### Area(s) with issue?

The bug also exists in RCT2 (vanilla)

### Steps to reproduce

1.place rapids in the water where boats roam and observe  that they then do not re dock to let off at the ride station exit
2. 
3.

**Comment by ocalhoun6** [comment]
Are you talking about "rapids" as in a custom scenery object, or "rapids" as in a section of rapids track from the river rapids ride?

(For an interim workaround in either case, you could place extra track on your boat hire ride before the station platform -- free roaming boats will dock with that track instead of the end of the platform, and you can place the boat hire track so that it leads to an open space where boats are able to go. You can then make the track invisible using the tile inspector, so it looks like boat hire boats smoothly go over the rapids and go to their station.)

**Comment by jamtraks** [comment]
I am referring to free roam boats such as rowing boats and bumper boats. I have a few set ups in a few of my parks, with no lead-track, they free roam fine with no rapid scenery in the water. If i place the rapid object scenery in the water where the boats free-roam and near the undocking exit, the boats don't lead into the exit to undock. The rapid object file itself prevents the boats from undocking. I've tested these boat rides with and  without the rapid object in the water  with the rapid object in the water they don't undock without it the do. I was told by another person here that the rapid object does in fact prevent the boats from undocking.

**Comment by duncanspumpkin** [comment]
To me it sounds like you want a flag that custom scenery can set that means boats ignore it when pathing. You would need to of course modify your custom object to add the flag but I don't see any major harm in doing this.

**Comment by jamtraks** [comment]
Yes, i want to be able to null the blocking of boats by the object file "A white water". Or i think the blocking of boat redocking by that object file should be permanently removed from the game somehow.

**Comment by duncanspumpkin** [comment]
Could you provide the object in question

**Comment by jamtraks** [comment]
"A white rapid" or "A white rapids"

**Comment by jamtraks** [comment]
Maybe there is a different white-water-rapids object file (named something else other than "a white water") that is visually identical to "a white water" but does not block the redocking of free roam boats, and i am only using the wrong rapids object file. 
I haven't searched for or checked for any other white-water-rapids object files though.
